packages/blockchain-wallet-v4-frontend/src/scenes/Login/EnterPassword/index.tsx

```diff
@@ -224,35 +226,37 @@ const EnterPassword = (props: Props) => {
         </WrapperWithPadding>
         <SignUpLink />
       </FormWrapper>
-      <MobileAuthSideWrapper>
-        <TextColumn>
-          <QRCodeWrapper value={qrData} size={150} showImage />
-          <Text
-            color='grey900'
-            size='14px'
-            weight={600}
-            lineHeight='1.25'
-            style={{ marginBottom: '8px' }}
-          >
-            <FormattedMessage
-              id='scenes.login.wallet.mobile_app_login.title'
-              defaultMessage='Or Log in with Mobile App'
-            />
-          </Text>
-          <Text color='grey900' size='12px' weight={500} lineHeight='1.5'>
-            <FormattedMessage
-              id='scenes.login.wallet.mobile_login.description.ios'
-              defaultMessage='<b>iOS</b> - Tap the Menu button at the top left corner of the app to reveal Web Log In option.'
-            />
-          </Text>
-          <Text color='grey900' size='12px' weight={500} lineHeight='1.5'>
-            <FormattedMessage
-              id='scenes.login.wallet.mobile_login.description.android'
-              defaultMessage='<b>Android</b> - Tap the QR code icon at the top right corner of the app.'
-            />
-          </Text>
-        </TextColumn>
-      </MobileAuthSideWrapper>
+      {!isMobile() && (
+        <MobileAuthSideWrapper>
```

Comment:
(minor): this name is a bit confusing, logic said show in case that it's not mobile and we then render `MobileAuthSideWrapper`